src/assets/image/sidebar/index.ts

```diff
@@ -1,5 +1,8 @@
+export { default as CircleQuestionIcon } from "@assets/image/sidebar/CircleQuestion.svg?react";
 export { default as ConnectionsIcon } from "@assets/image/sidebar/Connections.svg?react";
 export { default as DashboardIcon } from "@assets/image/sidebar/Dashboard.svg?react";
+export { default as FileIcon } from "@assets/image/sidebar/File.svg?react";
```

Comment:
Please mention the source of each icon (link to the figma)

Reply:
added